components/esp_websocket_client/examples/autobahn-testsuite/config/fuzzingserver.json

```diff
@@ -0,0 +1,15 @@
+{
+   "url": "ws://0.0.0.0:9001",
+   "options": {
+      "failByDrop": false
+   },
+   "outdir": "/reports",
+   "webport": 8080,
+   "cases": ["*"],
+   "exclude-cases": [
+      "9.*",
+      "12.*",
```

Comment:
Compression will be added once we introduce the extension, right? Does it make sense to have the performance suite configuration to be available for when we create a release MR?